### PR TITLE
OP-897 limit password retries

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-897

### CODE REVIEW

File: sql/step_04_all_following_steps.sql [hunk 1]
```diff
@@ -83,4 +83,4 @@ source step_86_permissions_and_grouppermission.sql;
 source step_87_alter_table_opd.sql;
 source step_88_alter_table_patient.sql;
 source step_89_alter_tables_innodb.sql;
-
+source step_90_login_retries.sql;
```
Comment (mwithi) [inline sql/step_04_all_following_steps.sql:86]: this step should be included also in Dockerfile-ohdb

File: src/main/java/org/isf/generaldata/GeneralData.java [hunk 2]
```diff
@@ -94,6 +94,8 @@ public final class GeneralData extends ConfigurationProperties {
 
 	public static boolean STRONGPASSWORD;
 	public static int STRONGLENGTH;
+	public static int PASSWORDTRIES;
+	public static int PASSWORDLOCKTIME;
```
Comment (mwithi) [inline src/main/java/org/isf/generaldata/GeneralData.java:98]: These new parameters should be added also in settings.properties.dist in "gui"

File: sql/step_04_all_following_steps.sql (hunk 1 above)
Comment (dbmalkovsky) [reply sql/step_04_all_following_steps.sql:86]: Done

File: src/main/java/org/isf/generaldata/GeneralData.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/generaldata/GeneralData.java:98]: Done; thanks

**Comment by dbmalkovsky** [comment]
Good catch; the code was updating the record in the database with the number of failures but checking again the local object that wasn't refreshed from the value in the db.   Fixed.

**Review by mwithi — APPROVED** [review]

**Pull request merged**